client/my-sites/plugins/plugin-details-sidebar/index.jsx

```diff
@@ -15,7 +14,39 @@ const PluginDetailsSidebar = ( props ) => {
 				{ translate( 'Plugin details' ) }
 			</div>
 			<div className="plugin-details-sidebar__plugin-details-content">
-				{ Boolean( active_installs ) && (
+				{ /* Needs to check for woocommerce dependencies */ }
+				{ isMarketplaceProduct && (
+					<PluginDetailsSidebarUSP
+						title={ translate( 'Your store foundations' ) }
+						description={ translate(
+							'This plugin requires the WooCommerce plugin to work. If you do not have it installed, it will be installed automatically for free.'
+						) }
+						first
+					/>
+				) }
+				{ /* Needs to check for demo_url */ }
+				{ isMarketplaceProduct && (
+					<PluginDetailsSidebarUSP
+						title={ translate( 'Try it before you buy it' ) }
+						description={ translate(
+							'Take a look at the posibilities of this plugin before your commit.'
+						) }
+						links={ [ { href: 'demo', label: translate( 'View live demo' ) } ] }
+					/>
+				) }
+				{ /* Needs to check for documentation_url */ },
+				{ isMarketplaceProduct && (
+					<PluginDetailsSidebarUSP
+						title={ translate( 'Support' ) }
+						description={ translate( 'Handled by WooCommerce.' ) }
+						last
+						links={ [
+							{ href: 'docs', label: translate( 'View documentation' ) },
+							{ href: 'https://automattic.com/privacy/', label: translate( 'See privacy policy' ) },
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **102** times:
`translate( 'privacy-policy' )` *ES Score: 6*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/plugin_details_sidebar_items)